docs/semgrep-ci/configuration-reference.md

```diff
@@ -19,6 +19,58 @@ While environment variables are the preferred way to configure Semgrep CI, pass
 SEMGREP_RULES="p/security-audit p/secrets"
 ```
 
+## Suppressing blocking comments or errors
+
+Semgrep in CI can block a pull request (PR) or a merge request (MR) from being merged. This block can be caused by Semgrep's blocking comment or internal error of Semgrep. One option to avoid blocked PRs and MRs is to change configuration of blocking comments in the rule board of Semgrep App. Another option that Semgrep in CI allows you is to disable blocking comments without changing the setup of rules in the rule board.
```

Comment:
I think we can just change this to something like:

```
Semgrep will exit with a non-zero exit code when there are blocking findings (exit code 1) or there is any issue in running semgrep (exit code 2 and above). Most CI providers block PR/MRs on non-zero exit codes. If you want your CI provider to not block a PR/MR due to semgrep failures you can setup semgrep below to return a non-zero exit code if there are any issues but still have semgrep exit with code 1 on blocking findings
```

Reply:
@brendongo This helps me better understand, thank you!

Reply:
Thanks! @brendongo Not sure about `if there are any issues but still have semgrep exit with code 1 on blocking findings`. Is this describing `semgrep ci || [ $? != 1 ]`?

Changed to:

```
Most CI providers block pull requests (PRs) or merge requests (MRs) on non-zero exit codes. Semgrep exits with a non-zero exit code (exit code 1) when it reports a blocking finding. Other non-zero exit codes that also block your PRs and MRs signify an issue in running Semgrep (exit code 2 and above). For more information about specific Semgrep exit codes see [CLI reference](../../cli-reference/#exit-codes).

Configure the behavior of Semgrep in CI to pass on blocking findings or internal Semgrep errors blocking your pipeline by using the following options in your YAML configuration file: 
```

Reply:
@brendongo implemented. Let me know if the text is fine now!